school/models/school.py

```diff
@@ -486,20 +515,51 @@ def create(self, vals):
 
 
 class SubjectSubject(models.Model):
-    """Defining a subject"""
+    """Defining a subject """
 
     _name = "subject.subject"
     _description = "Subjects"
 
+    def _compute_student_subject(self):
+        """Compute student of done state"""
+        self.student_ids = False
+        subject_list = []
+        standard_obj = self.env["school.standard"]
+        standard_id = standard_obj.search(
```

Comment:
you can directly use looping on search, no need to assign the result of search in "standard_id"